src/managed/CommonManaged.props

```diff
@@ -15,15 +15,6 @@
     <PackageThirdPartyNoticesFile>$(RepoRoot)THIRD-PARTY-NOTICES.TXT</PackageThirdPartyNoticesFile>
   </PropertyGroup>
 
-  <PropertyGroup Condition="'$(StabilizePackageVersion)' != 'true'">
```

Comment:
Why did you remove the stable version suffix workaround?

Reply:
Because this is handled by the root dir.props now.

https://github.com/dotnet/core-setup/blob/bb6dac8404513d5ac5a1a216e362ec2ff5ea8ef5/dir.props#L74-L76

Technically, this change should be made in `master` as well.

Reply:
I see. Interestingly it doesn't show up when I do "view all" diff it only shows up on the individual commit which is a little odd.

Reply:
That's because this addition came in a cherry-picked commit, and then I removed it as part of "fixing up changes from master".  Originally this was needed because the "managed" projects didn't import the root dir.props.  So this change was needed.  But then later I changed it so these projects do import the root dir.props.  That makes this code unnecessary now.

I'll port this change to master as well, since it is unnecessary there.